src/subscription/Captcha.ts

```diff
@@ -17,12 +17,18 @@ import { uint8ArrayToBase64 } from "@tutao/tutanota-utils"
  * @returns {string} HH:MM if parsed, null otherwise
  */
 export function parseCaptchaInput(captchaInput: string): string | null {
-	if (captchaInput.match(/^[0-2]?[0-9]:[0-5]?[05]$/)) {
+	if (captchaInput.match(/^[0-2]?[0-9]:[0-5]?[0-9]$/)) {
```

Comment:
I think before it explicitly only matched minutes in 5 min intervals. Did you change this intentionally?

Reply:
I see that tests actually expected 1 min intervals. I would still check with @armhub if this was intended.

Reply:
Hey, I did not change it to make the tests pass. This is just because of what the issue was complaining about:

> When trying to solve a captcha and entering minutes outside of a 5 minute interval (11 instead of 10) I get an error message saying "Please enter the time in hours and minutes." which is misleading.

The old regex rejects times that are not 5 minute intervals, thus you get an error stating that the time is malformatted, not that it is wrong.

The solution I found, without changing the error to something complex such as "Please enter the time in hours and minutes, and only in 5 minute intervals." is to allow all correctly formatted times, not just 0/5 minute times, but check them locally.

Reply:
I suppose another alternative is to not return null, but to return a constant in some enum that we can check against, or throw an error of some kind that we can catch.

I really don't care about the tests checking against *1 to *4 and *6 to *9 minutes. As far as I know, we will never support these.

Reply:
I see, I understand now, thank you for explaining!